``pytest.raises`` now raises ``TypeError`` when receiving an unknown keyword argument.
